src/page-templates/content-type/products-section.tsx

```diff
@@ -28,39 +29,75 @@ const ProductsSection: React.FunctionComponent<ProductsSectionProps> = ({
             </TypographyScale>
 
             <Grid
-                columns={[3]}
+                columns={items.length}
                 sx={{
-                    gap: ['inc30', null, 'inc40'],
+                    display: ['grid', null, 'none'],
                     marginBottom: 'inc40',
+                    marginTop: ['inc30', null, null, 'inc40'],
+                    paddingBottom: ['inc30', null, '0'],
+                    gap: ['inc40', null, null, 'inc40'],
+                    overflow: ['auto', null, 'visible'],
+                    mx: [`-${theme.space.inc40}`, null, 'unset'],
+                    px: ['inc40', null, 'unset'],
                 }}
             >
-                {items
-                    .slice(0, itemsToDisplay)
-                    .map(({ titleLink, imageString, cta, links }) => {
-                        const image = imageString ? (
-                            <BrandedIcon name={imageString} />
-                        ) : null;
-                        return (
-                            <ShowcaseCard
-                                key={titleLink.text}
-                                alignment="left"
-                                image={image}
-                                titleLink={titleLink}
-                                cta={cta}
-                                sx={{ width: '100%' }}
-                                links={links}
-                            />
-                        );
-                    })}
+                {items.map(({ titleLink, imageString, cta, links }) => {
+                    const image = imageString ? (
+                        <BrandedIcon name={imageString} />
+                    ) : null;
+                    return (
+                        <ShowcaseCard
+                            key={titleLink.text}
+                            alignment="left"
+                            image={image}
+                            titleLink={titleLink}
+                            cta={cta}
+                            sx={{
+                                boxSizing: 'border-box' as 'border-box',
+                                width: ['66vw', null, 'auto'],
+                            }}
+                            links={links}
+                        />
+                    );
+                })}
             </Grid>
-            {items.length > 3 && (
-                <Link
-                    onClick={() => setItemsExpanded(!itemsExpanded)}
-                    sx={{ display: 'block', mx: 'auto' }}
+
+            <div sx={{ display: ['none', null, 'block'] }}>
+                <Grid
+                    columns={[3]}
+                    sx={{
+                        gap: ['inc30', null, 'inc40'],
+                        marginBottom: 'inc40',
+                    }}
                 >
-                    {itemsExpanded ? 'Less' : 'More'} Products
-                </Link>
-            )}
+                    {items
```

Comment:
which one is this ticket?

Reply:
https://jira.mongodb.org/browse/DEVHUB-1219